src/registrar/templates/home.html

```diff
@@ -41,7 +41,16 @@ <h2>Domains</h2>
           <th data-sortable scope="col" role="columnheader">Domain name</th>
           <th data-sortable scope="col" role="columnheader">Expires</th>
           <th data-sortable scope="col" role="columnheader">Status</th>
-          <th scope="col" role="columnheader"><span class="usa-sr-only">Action</span></th>
+          <th 
+            scope="col" 
+            role="columnheader" 
+            {% if has_deletable_applications %} class="action-col-custom-width" {% else %} class="width-15" {% endif %}
+          >
+            <span class="usa-sr-only">Action</span>
+          </th>
+          {% if has_deletable_applications %}
+          <th></th>
+          {% endif %}
```

Comment:
Technically we don't need to add this, just up the width -- but this seems to work better with scaling. Its the most reliable solution I found for it so far, but I'd be open to other suggestions

Reply:
Why add this at all?